.gitignore

```diff
@@ -33,3 +33,5 @@ logs
 
 tests/ef_tests/test_data
 !tests/ef_tests/test_data/.gitkeep
+
+bin/
```

Comment:
new line (vs code config is

"files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,